normandy/control/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+from django.conf.urls import url, include
+
+from normandy.control import views as control_views
+from django.contrib.auth.views import login, logout_then_login
+
+urlpatterns = []
+
+if settings.ADMIN_ENABLED:
+  urlpatterns += [
+      url(r'^control/', include([
+          url(r'^$', control_views.IndexView.as_view(), name='index'),
```

Comment:
@Osmose @mythmon @rehandalal yay or nay on URL namespacing? I can revert if we don't like. https://github.com/mozilla/normandy/pull/95/commits/8b23cb17ecc53ca9ec830c864c11ad5950d0333b